src/test/java/com/rocketden/main/socket/RoomSocketTests.java

```diff
@@ -161,7 +169,6 @@ public void socketReceivesMessageOnHostChange() throws Exception {
         assertEquals(expected.getUsers(), actual.getUsers());
     }
 
-    @Test
     public void socketReceivesMessageOnSettingsChange() throws Exception {
```

Comment:
nit: Re-add as test

Reply:
Or if it's still broken for some reason we can pair program to try to figure it out, or leave a TODO comment to make sure to address it in the next PR

Reply:
Shoot sorry, I didn't realize I removed this test -- re-added and it works perfect! 🙂